### ISSUE TITLE
Linkedin share is terrible

### ISSUE THREAD

**Issue by xelhark** [720kb/angular-socialshare#27]
Hey I noticed that linkedin share doesn't work if the url is encoded (go figure... ) I know it's not your fault and that URL should be encoded, but it just doesn't work.

I added an option (which is currently only active for linkedIn) which adds the option of NOT encoding the URL when sharing. If you're interested I'm adding a pull request.

**Comment by xelhark** [comment]
Ok, after a lot of digging and fighting with the customer support (who didn't even help me after all), I noticed that some services do NOT handle the hashbang gracefully, and so you will be left with just the home page version, because everything that comes after the hashbang will not work. (LinkedIn is one of these for example, but Slack has the same problem).

I fixed it by using HTML5 mode, even though I lost some compatibility. I hope this helps someone if it were to fall into the same problem.
